src/util.js

```diff
@@ -250,10 +250,11 @@ export function isInDialog(event, container = null) {
         return !!findClosestElWithClass(event.target, CLASS_ANNOTATION_POPOVER);
     }
 
-    const annotatorLabelEl = container.querySelector('.ba-annotator-label');
-    const commentListEl = container.querySelector('.ba-comment-list');
-    const controlsEl = container.querySelector('.ba-action-controls');
-    return isInElement(event, annotatorLabelEl) || isInElement(event, commentListEl) || isInElement(event, controlsEl);
+    return (
+        !!findClosestElWithClass(event.target, 'ba-annotator-label') ||
```

Comment:
This method will check if the event falls within the element specified by the provided classname or any of it's children rather than checking event target x/y coordinates vs the specific divs above

Reply:
What's the advantage of that over isInElement?

Reply:
It will more cleanly catch cases where this was triggered via a touch event. This is why on touch devices, clicking in the center of the text area would often cause the entire popover to close